content/post/how-to-be-bleeding-edge-c++-proposals.md

```diff
@@ -0,0 +1,98 @@
++++
+categories = []
+date = "2015-09-05T23:57:10-07:00"
+description = ""
+keywords = []
+title = "How to be bleeding edge – C++ proposals that'll change the way you code"
+
++++
+
+In this article, I'll go over a few of the most recently implemented C++ features and extensions in GCC and Clang. Some might just change the face of your code forever. At the very least, demonstrating your bleeding edge knowledge in interviews will be sure to get you some points.
```

Comment:
Might be worth using this as the description as well. In the absence of a description, the first X words are thrown together.